### PR TITLE
dhall 1.33.1 dhall-bash 1.0.31 dhall-json 1.7.0 dhall-lsp-server 1.0.8 dhall-yaml 1.2.0

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

- `dhall 1.33.1`
- `dhall-bash 1.0.31`
- `dhall-json 1.7.0`
- `dhall-lsp-server 1.0.8`
- `dhall-yaml 1.2.0`

### CODE REVIEW

File: Formula/dhall-yaml.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 class DhallYaml < Formula
   desc "Convert between Dhall and YAML"
   homepage "https://github.com/dhall-lang/dhall-haskell/tree/master/dhall-yaml"
-  url "https://hackage.haskell.org/package/dhall-yaml-1.1.0/dhall-yaml-1.1.0.tar.gz"
-  sha256 "72919ce641af46b6f2e8884ad358f545b58023682f65b2b3d1d1499974fc9c1a"
+  url "https://hackage.haskell.org/package/dhall-yaml-1.2.1/dhall-yaml-1.2.1.tar.gz"
```
Comment (miccal) [inline Formula/dhall-yaml.rb:4]: I think this should be for `1.2.0`, not `1.2.1`.

Comment (chenrui333) [reply Formula/dhall-yaml.rb:4]: yeah, you are right.

**Comment by chenrui333** [comment]
@miccal thanks for calling it out. :)

**Comment by miccal** [comment]
You're welcome @chenrui333

File: Formula/dhall-yaml.rb [hunk 2]
```diff
@@ -24,7 +23,6 @@ def install
   test do
     assert_match "1", pipe_output("#{bin}/dhall-to-yaml-ng", "1", 0)
     assert_match "- 1\n- 2", pipe_output("#{bin}/dhall-to-yaml-ng", "[ 1, 2 ]", 0)
-    assert_match "x: 1\ny: 2", pipe_output("#{bin}/dhall-to-yaml-ng", "{ x = 1, y = 2 }", 0)
```
Comment (chenrui333) [inline Formula/dhall-yaml.rb:27]: have some local installation issue for `dhall-yaml`, I will come back for this.

**Comment by chenrui333** [comment]
cborg bug tracked in here, https://github.com/well-typed/cborg/issues/242

**Comment by chenrui333** [comment]
Upstream issue, closing the PR for now.

**Review by SMillerDev — APPROVED** [review]

**Comment by chenrui333** [comment]
Nice, thanks @alebcay!

**Pull request closed without merging**